pages/common/cargo-remove.md

```diff
@@ -0,0 +1,36 @@
+# cargo remove
+
+> Remove dependencies from a `Cargo.toml` manifest file.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-remove.html>.
+
+- Remove one or more dependencies from the `Cargo.toml` manifest:
+
+`cargo remove {{dependency_name}}`
+
+- Remove a build dependency:
+
+`cargo remove --build`
```

Comment:
I think all `remove` commands do expect a `dependency_name`, so i would request to add it as appropriate.
I tried to run the `cargo remove` on a rust project and it expects a `dependency_name`.

Reply:
Okay my bad, I will fix it ASAP.